src/univalue/include/univalue.h

```diff
@@ -25,6 +25,9 @@ class UniValue {
         typ = initialType;
         val = initialStr;
     }
+   UniValue(size_t val_) {
```

Comment:
@daira I created a pull request to @str4d UniValue tree, once it's accepted I'll merge them in, so this indentation issue won't be present.